airbyte_cdk/sources/declarative/parsers/model_to_component_factory.py

```diff
@@ -2782,29 +2787,29 @@ def _get_download_retriever() -> SimpleRetrieverTestReadDecorator | SimpleRetrie
                 parameters={},
             )
 
-        decoder = (
+        operational_decoder = (
```

Comment:
What does "operational" means here?

Reply:
I've reverted this change, it was probably not worth it.